escrow/validation.go

```diff
@@ -206,3 +214,15 @@ func bigIntToBytes(value *big.Int) []byte {
 func bytesToBigInt(bytes []byte) *big.Int {
 	return (&big.Int{}).SetBytes(bytes)
 }
+
```

Comment:
Lets make it a generic error message instead of specifying that the service is under curation